packages/zowe-explorer/src/job/actions.ts

```diff
@@ -162,16 +162,28 @@ export const focusOnJob = async (jobsProvider: IZoweTree<IZoweJobTreeNode>, sess
 export async function modifyCommand(job: Job) {
     try {
         const command = await vscode.window.showInputBox({
-            prompt: localize("modifyCommand.command.prompt", "Modify Command"),
+            prompt: localize("jobActions.modifyCommand.command.prompt", "Modify Command"),
         });
         if (command !== undefined) {
-            const response = await zowe.IssueCommand.issueSimple(job.getSession(), `f ${job.job.jobname},${command}`);
-            vscode.window.showInformationMessage(
-                localize("modifyCommand.response", "Command response: ") + response.commandResponse
-            );
+            const commandApi = ZoweExplorerApiRegister.getInstance().getCommandApi(job.getProfile());
+            if (commandApi) {
+                const response = await ZoweExplorerApiRegister.getCommandApi(job.getProfile()).issueMvsCommand(
+                    `f ${job.job.jobname},${command}`
+                );
+                vscode.window.showInformationMessage(
+                    localize("jobActions.modifyCommand.response", "Command response: ") + response.commandResponse
+                );
+            }
```

Comment:
@crawr do you know how we checked for successful command before for the popup?  As you can see here, I used the same output as before, just calling the API vs zowe cli directly.

Reply:
Yes, it looks like we just pass the message from the API itself. This could be part of the broader discussion on the error handling we want to implement in the future.